dhcpv6/dhcpv6_test.go

```diff
@@ -150,5 +150,21 @@ func TestNewReplyFromRebind(t *testing.T) {
 	require.Equal(t, rep.Type(), REPLY)
 }
 
+func TestNewSolicit(t *testing.T) {
+	s, err := NewSolicit()
+	require.NoError(t, err)
+
+	require.Equal(t, s.Type(), SOLICIT)
+
+	oroOption := s.GetOneOption(OPTION_ORO)
+	require.NotNil(t, oroOption)
+	oro, ok := oroOption.(*OptRequestedOption)
+	opts := oro.RequestedOptions()
+	require.True(t, ok)
+	require.Contains(t, opts, DNS_RECURSIVE_NAME_SERVER)
+	require.Contains(t, opts, DOMAIN_SEARCH_LIST)
+	require.Equal(t, len(opts), 2)
```

Comment:
after adding the empty CID we also need to test that it exists and has a zeroed LL addr